pkg/datastore/sql_struct.go

```diff
@@ -44,6 +44,7 @@ const (
 
 // Maps DB table name to booleans showing if the table supports multitenancy and revisioning.
 var schemaMap = map[string]map[string]bool{}
+var tableNameMap = map[string]string{}
```

Comment:
Could you describe in the comments what the key and value are gonna be?

Reply:
Done @jeyhun  any other comments?